Tech and Fin analysis/TA.py

```diff
@@ -12,12 +12,14 @@ def sma(ticker):
     df[['SMA']].plot(figsize=(12,12))
     # plt.show()
     return plt
+    return plt.show()

```

Comment:
**Issue**: Incorrect indentation after control structures

**Reason**: The line 'return plt.show()' is incorrectly indented with 4 spaces instead of being aligned with the function body.

**Suggestion**: Remove the extra space at the beginning of the line to align it with the function body.